src/pages/learn/[id].tsx

```diff
@@ -359,9 +375,21 @@ function LearnDetail() {
       <NavigationBar />
       <StLearnDetail>
         <ImageDiv onClick={() => router.push(prevLink)} src={icXButton} className="close" layout="fill" alt="x" />
-        {videoData?.names && (
-          <StScriptTitleContainer>
-            {videoData.names.map(({ id, name }, i) => (
+        <StScriptTitleContainer>
+          {videoData?.haveGuide && (
+            <StGuideTitle isGuide={isGuide} onClick={() => !isGuide && setIsGuide((prev) => !prev)}>
+              <p>스피치 가이드</p>
+              <ImageDiv
+                className="guide-info"
+                src={icSpeechGuideInfo}
+                alt="speech-guide-info"
+                onMouseOver={() => isGuide && setIsGuideOver((prev) => !prev)}
+                onMouseOut={() => isGuide && setIsGuideOver((prev) => !prev)}
```

Comment:
```suggestion
                onMouseOver={() => setIsGuideOver(true)}
                onMouseOut={() => setIsGuideOver(false)}
```

여기 isGuide가 필요한 이유가 있을까요?
그리고 마우스 호버 시에는 항상 true, 아니면 false로 세팅해야 해서 prev 안 써도 될 것 같은데 어떻게 생각하시나요?

Reply:
해당 툴팁은 스피치 가이드 페이지에서만 떠야 하기 때문에 isGuide가 필요합니다!
prev는 수정하였습니다 :D